arch/arm/src/stm32/stm32_sdio.c

```diff
@@ -172,9 +172,14 @@
 #define SDIO_CMDTIMEOUT          (100000)
 #define SDIO_LONGTIMEOUT         (0x7fffffff)
 
-/* Big DTIMER setting */
+/* DTIMER setting
+ *
+ * Assuming Max timeout in bypass 48 Mhz
+ *
+ */
 
-#define SDIO_DTIMER_DATATIMEOUT  (0x000fffff)
+#define IP_CLCK_FREQ 48000000ll
```

Comment:
```suggestion
#define IP_CLCK_FREQ UINT32_C(48000000)
```
This macro from `inttypes.h` is not much widespread, but it is a more portable way for defining minimum-width integer constants.

Reply:
Hmm, It is not intuitive or obvious.
Do you object to leaving it as is in this acrh specific file?

Reply:
> Hmm, It is not intuitive or obvious.

I don't agree with not being intuitive. The `ll` suffix is implementation-dependent, so you need prior knowledge on this. Meanwhile, using `UINT32_C` these details are abstracted, the developer just needs to ensure that this integer constant is at least 32-bit wide.

By the way, `IP_CLCK_FREQ` will be attributed to a `uint32_t` variable, so adding the `ll` suffix you are prone to type conversion. The correct for the `armv7-m` architecture would be `ul`.

> Do you object to leaving it as is in this acrh specific file?

No objection at all. But I believe that using the macros from `inttypes.h` is less error-prone.

Reply:
Done! I had to fix the CI fails and I can not disagree with your sound logic!